resources/assets/sass/layout/_footer.scss

```diff
@@ -142,3 +142,24 @@ footer {
     display: inline-block;
   }
 }
+
+#back-to-top {
+  display: none;
+  width: 60px;
+  height: 60px;
+  text-indent: -9999px;
+  position: fixed;
+  z-index: 999;
+  right: 15px;
+  bottom: 40px;
+  background: #000 url("/img/up-arrow.png") no-repeat center 43%;
+  -webkit-border-radius: 30px;
+  -moz-border-radius: 30px;
+  border-radius: 30px;
+  opacity: 0.3;
+
+  &:hover {
+    opacity: 0.5;
+    background-color: #000;
+  }
+}
```

Comment:
brakuje nowej linii na koncu pliku (http://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file)

ja korzystam z phpstorm i mozna ustawic, aby z automatu dodawal nowa, pusta linie. do niedawna tez nie wiedzialem o tym, ale rozne narzedzia sprawdzajace poprawnosc kodu "krzycza", ze plik nie zawiera pustej linii na koncu